src/client/src/apps/SimpleLauncher/tools.ts

```diff
@@ -71,12 +71,20 @@ export async function open(
 }
 
 async function launchLimitChecker(config: ApplicationConfig) {
-  const app = fin.Application.wrap({ uuid: config.name })
-  fin.desktop.System.launchExternalProcess({
-    alias: 'LimitChecker',
-    listener(result) {
-      console.log('the exit code', result.exitCode)
+  const app = fin.Application.wrap({uuid: config.name})
+  console.log ("app", app);
```

Comment:
what is this for? can we remove this please or make this log line more meaningful?